.gitignore

```diff
@@ -34,4 +34,7 @@ build.metadata
 *.lock
 *.tgz
 /**/nbproject/
-.ci/openshift-ci/build-root/scripts/InstallCert/jssecacerts
\ No newline at end of file
+.ci/openshift-ci/build-root/scripts/InstallCert/jssecacerts
+ejb-txn-remote-call/server1
+ejb-txn-remote-call/server2
+ejb-txn-remote-call/server3
```

Comment:
As I usually unzip the server somewhere else (like `ejb-txn-remote-call/../..`) I wouldn't pollute `.gitignore` with stuff related to ejb-txn-remote-call

Reply:
ok